cli/template/base/package.json

```diff
@@ -7,7 +7,13 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "check": "next lint && tsc --noEmit",
+    "preview": "next build && next start",
+    "typecheck": "tsc --noEmit",
+    "format:write": "prettier --write \"**/*.{ts,tsx,js,jsx,mdx}\" --cache",
+    "format:check": "prettier --check \"**/*.{ts,tsx,js,jsx,mdx}\" --cache"
```

Comment:
Not all templates gets prettier installed, only if you have tailwind iirc so this wouldn't work ootb

Reply:
Thank you for the feedback! Just to clarify, would you like me to remove the scripts entirely, or would you prefer that I integrate them through [`tailwind.ts`](https://github.com/programming-with-ia/create-t3-app/blob/main/cli/src/installers/tailwind.ts) instead, similar to [this example](https://github.com/programming-with-ia/create-t3-app/blob/1567a481d51eff4fda60dae094775d76ac273863/cli/src/installers/drizzle.ts#L79C1-L88C5)? I'm happy to make the adjustments based on your preference.

Reply:
modify `cli\src\installers\tailwind.ts`
![image](https://github.com/user-attachments/assets/5bf7dffd-904d-4af2-b2bc-ce64656d618e)

Reply:
yea similar to how the db installers adds the db:push scripts etc this can for now be handled in the tailwind installeer if/until there're more formatting options

Reply:
also for "typecheck": "tsc --noEmit" script

Reply:
@juliusmarminge 

Done ✅

Reply:
needs formatting

Reply:
> needs formatting

Okay

Reply:
you need to run `pnpm format` in root. there are formatting issues in the PR

Reply:
@juliusmarminge Done

Reply:
[Is it necessary to run `changeset add --empty` or `changeset add`?](https://github.com/t3-oss/create-t3-app/actions/runs/11364580755/job/31610905592)

Reply:
Yes. Read the contributing guide (you checked the box when creating the or ☺️)

Reply:
> Yes. Read the contributing guide (you checked the box when creating the PR ☺️)

🙏 🫡😔